src/service/viewer-impl.js

```diff
@@ -1005,7 +1005,7 @@ export class Viewer {
       return Promise.reject(getChannelError());
     }
     return this.messagingReadyPromise_.then(() => {
-      return this.sendMessageUnreliable_(eventType, data, awaitResponse);
+      return this.messageDeliverer_(eventType, data, awaitResponse);
```

Comment:
Basically, the message is NOT guaranteed to be delivered. It doesn't wait for the messaging to be established and may or may not be queued.